more/free-programming-cheatsheets.md

```diff
@@ -82,6 +82,7 @@
 * [Git Cheat Sheet](http://git.jk.gs) - Jan Krüger (PDF, SVG)
   * [Git Cheat Sheet](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet.pdf) (PDF)
   * [Git Cheat Sheet - extended](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet-v2.zip) (PDF)
+* [Git Cheat Sheet](https://downloads.git-tower.com/content/cheat-sheet-bundle.zip) - Tower (PDF)
```

Comment:
is there a link that isn't zipped? Tower is an author?

Reply:
Frontend: https://www.git-tower.com/blog/git-cheat-sheet/

Reply:
> is there a link that isn't zipped? Tower is an author?

No, they only have a zip link. It is only possible to add direct .pdf links?

Reply:
> > is there a link that isn't zipped? Tower is an author?
> 
> No, they only have a zip link. It is only possible to add direct .pdf links?

Not is limited, I thought

There are very PDF cheatsheets about this company inside. This includes Git, VSCode, Shell...

Reply:
It's not like we need another git cheatsheet. The thing about zips is it's easy to hide something bad inside.  

Weill close, feel free to reopen if the content is compelling.